miniupnpd/Makefile.linux

```diff
@@ -23,13 +23,19 @@
 #
 CONFIG_OPTIONS ?= $(cat .configure.cache)
 CONFIG_OPTIONS += --firewall=iptables
+
+# Make sucks, prove me wrong
+, := ,
+ifeq ($(filter -D_FORTIFY_SOURCE=% -Wp$(,)-D_FORTIFY_SOURCE=%,$(CPPFLAGS) $(CFLAGS)),)
+CPPFLAGS += -D_FORTIFY_SOURCE=2
+endif
+CPPFLAGS += -D_GNU_SOURCE
```

Comment:
I won't say it sucks, I'll say it has some quirks :laughing: 

I don't remember in which project I have a variable which contains a single space :sweat_smile: